include/deal.II/base/function.templates.h

```diff
@@ -26,11 +26,6 @@
 
 DEAL_II_NAMESPACE_OPEN
 
-
-template <int dim, typename Number>
-const unsigned int Function<dim, Number>::dimension;
```

Comment:
Isn't the definition/definition of this already in `function.h`? That is why I removed this.